tutorials/sphinx_tuto/tensordict_shapes.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""
+Manipulating the shape of a TensorDict
+======================================
+**Author**: `Vincent Moens <https://github.com/vmoens>`_
```

Comment:
Put your name there mate!

Reply:
Oops, I copied a different tutorial as a starting point and missed that.